src/Compilers/CSharp/Portable/Symbols/PublicModel/EventSymbol.cs

```diff
@@ -62,6 +62,14 @@ ITypeSymbol IEventSymbol.Type
             }
         }
 
+        IFieldSymbol? IEventSymbol.BackingField
+        {
+            get
+            {
+                return _underlying.AssociatedField.GetPublicSymbol();
```

Comment:
>_underlying.AssociatedField.GetPublicSymbol(); [](start = 23, length = 46)

Is this symbol returned by GetMembers API?

Reply:
No. Event backing fields are not, starting in 2011 judging by the comments that were left that hinted at removing property backing fields as well which has never happened.

Reply:
>> Is this symbol returned by GetMembers API?

> No. Event backing fields are not

I don't think I am comfortable with this inconsistency

---
In reply to: [532762816](https://github.com/dotnet/roslyn/pull/49659#discussion_r532762816) [](ancestors = 532762816)

Reply:
> I don't think I am comfortable with this inconsistency

What do you suggest? This new API is motivated largely as a way to ignore the underlying inconsistency from the perspective of analyzer writers.